src/pages/tutorials.tsx

```diff
@@ -0,0 +1,65 @@
+import algoliasearch from 'algoliasearch/lite'
+import { InstantSearch } from 'react-instantsearch-hooks-web'
+import { history } from 'instantsearch.js/es/lib/routers'
+import { useRouter } from 'next/router'
+
+import SidebarSidecarLayout, {
+	SidebarSidecarLayoutProps,
+} from 'layouts/sidebar-sidecar'
+import TutorialsLibraryView from 'views/tutorial-library'
+import { TutorialLibraryFilters } from 'views/tutorial-library/components/filters'
+import { INDEX_NAME } from 'views/tutorial-library/constants'
+import {
+	routerStateToSearchState,
+	searchStateToRouteState,
+} from 'views/tutorial-library/utils/router-state'
+
+const appId = process.env.NEXT_PUBLIC_ALGOLIA_APP_ID
+const searchClient = algoliasearch(appId, 'bf27a047ba263cba01ee9b4081965a1a')
+
+interface TutorialsLibraryPageProps {
+	layoutProps: Omit<SidebarSidecarLayoutProps, 'sidecarSlot' | 'headings'>
+}
+
+export default function TutorialsLibraryPage({
```

Comment:
Curious if we should make this `/tutorials/library` now 💭

Reply:
I think I'm gonna do it, we can adjust if we hear otherwise.